packages/hooks/tsconfig.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
if https://github.com/Developer-DAO/web3-ui/pull/25 goes in, then this file will probably get deleted too in favor of the root version